### PR TITLE
Gives medical cyborgs a siren

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
add: Medical Cyborgs now have a siren.
/:cl:

### CODE REVIEW

**Comment by GunHog** [comment]
Oh, my! This is both hilarious and terrible! I love it!

File: code/game/objects/items/robot/robot_items.dm [hunk 1]
```diff
@@ -327,6 +327,11 @@
 		cooldown = world.time + 600
 		log_game("[user.ckey]([user]) used an emagged Cyborg Harm Alarm in ([user.x],[user.y],[user.z])")
 
+/obj/item/clothing/head/helmet/justice/cyborg
+	name = "Ambulance Siren"
+	desc = "Emergency, get out of the way!"
+	icon_state = "megaphone"
```
Comment (GunHog) [inline code/game/objects/items/robot/robot_items.dm]: Have you tested this for weird 'you turn on the light' messages and potential sprite issues when toggling?

**Comment by Fox-McCloud** [comment]
🚔🚨:sound:wee-woo-wee-woo🔉🚨🚔

**Comment by kevinz000** [comment]
WEW WEW HUMAN HARM WEW WEW

File: code/game/objects/items/robot/robot_items.dm (hunk 1 above)
Comment (KorPhaeron) [inline code/game/objects/items/robot/robot_items.dm]: Good catch

**Comment by kevinz000** [comment]
It would be very, very cool if you could add siren sprites to the mediborg and make them turn on when this is activated.

**Comment by optimumtact** [comment]
Stop giving players a way to annoy others with noise :-1: from me

**Pull request closed without merging**